automatic/ssis-vs2019/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageArgs = @{
+    packageName   = $env:ChocolateyPackageName
+    softwareName  = 'Microsoft.DataTools.IntegrationServices*'
+    fileType      = 'exe'
+    silentArgs    = "/quiet /log `"$env:TEMP\$env:ChocolateyPackageName.$env:ChocolateyPackageVersion.log`""
+    validExitCodes= @(0)
+    url           = "https://ssis.gallerycdn.vsassets.io/extensions/ssis/sqlserverintegrationservicesprojects/3.0/1555036193942/Microsoft.DataTools.IntegrationServices.exe"
+    checksum      = "f96d1152d442a9ffa46c02d71dbeaed4afc09a9a4b25564d4ab927fd1be1d38c"
```

Comment:
`url` and `checksum` don't need to be set necessarily as they will be set by the update script after a new version is detected if version in `.nuspec` is not set

Reply:
Have changed it to "update". Empty string is not recognized... Will be updated with the first release.